packages/@guardian/source-foundations/jest.config.js

```diff
@@ -2,4 +2,5 @@
 module.exports = {
 	preset: 'ts-jest',
 	testEnvironment: 'node',
+	setupFilesAfterEnv: ['./src/utils/jest-matchers/index.ts'],
```

Comment:
could we move this to `packages/@guardian/source-foundations/lib`? the other things that live in `utils` are actually code that ships, whereas this is code that analyses that code.

appreciate the naming of these folders is not super clear, but we do have a pattern of using `libs` (e.g. at the root) for this type of thing, so it would be consistent, if not clear 🙃 

we should probably make this stuff a bit clearer subsequently

Reply:
moved!